test/performance/odp_timer_accuracy.c

```diff
@@ -1406,6 +1446,9 @@ int main(int argc, char *argv[])
 	}
 
 	odph_thread_join(thread_tbl, num_workers);
+
+	test_global->common_options = common_options;
```

Comment:
I'd perhaps move this closer to `test_global` reservation. E.g. after `memcpy(&test_global->opt, &test_opt, sizeof(test_opt_t));` line.